### ISSUE TITLE
Design the site

### ISSUE THREAD

**Issue by Alonski** [Alonski/CoinPanion#4]
Mobile First!
Colors?
Hamburger Menu?
Pages:
Home page: Explanation + Top users + Link to Create Profile
Create Profile:
User Profiles:
About Us:

**Comment by rhlsthrm** [comment]
I will create a Balsamiq mockup of the screens.

**Comment by Alonski** [comment]
Great :smile:

**Comment by Josephrp** [comment]
I'm commenting about general design here : 

Assure that it responds to the better practices for accessibility (colors for color blind, navigation for the blind, minimum sizes on all buttons etc. more info : https://www.w3.org/WAI/mobile/

just a suggestion, but this type of details are important....

**Comment by Alonski** [comment]
@Josephrp While I agree we should strive to be accessible this isn't a main concern for the MVP.
Definitely for anything later than Beta

**Comment by rhlsthrm** [comment]
Agreed, we will be using an out of box theme (Bootstrap/material) so we shouldn't really have to worry about button sizes and things like that.

**Comment by Josephrp** [comment]
May i suggest a CSS? at least do the utmost : i'm sure it's well regarded from the organisers.

Again, i have to insist here, if it's an afterthought its painful and expenssive, if it's there from the start then it makes sense, and folks will adopt it as a model for sure,,, there's no technical reason why the code shouldn't respect basic quality thresholds.
